packages/shared-component--icon/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@coopdigital/shared-component--icon",
+  "version": "1.0.0",
```

Comment:
Can you make this `0.1.0` or something below `1.0.0`?

Otherwise as soon as we publish it'll become `1.0.1` 😬